upgrade/steps.go

```diff
@@ -163,11 +163,11 @@ func UpgradeProviderVersion(
 			// We need to remove any patches to so we can cleanly pull the next upstream version.
 			step.Cmd("git", "reset", "HEAD", "--hard").In(&upstreamDir),
 			// Load patches into a branch.
-			step.Cmd("./upstream.sh", "checkout").In(&repo.root),
+			step.Cmd(".scripts/upstream.sh", "checkout").In(&repo.root),
```

Comment:
missing slash?

Reply:
Yup! Good catch! Guess the tests don't catch much in this project 🙈